frontend/src/routes/Governance/policies/Policies.tsx

```diff
@@ -559,7 +564,11 @@ export default function PoliciesPage() {
                 processing: t('policy.table.actions.enforcing'),
                 items: [...item],
                 emptyState: undefined, // there is always 1 item supplied
-                description: t('policy.modal.message.enforce'),
+                description:
+                  getInformOnlyPolicies(item) == ''
```

Comment:
If you only use this function for these, isn't it better to change the function to return boolean?  `hasInformOnly` something like this

Reply:
Oh yea, originally I was going to list out the names of the policies, but now the functionality is different so I will change that.